.ci/install_kata_kernel.sh

```diff
@@ -65,11 +65,12 @@ get_packaged_kernel_version() {
 }
 
 build_and_install_kernel() {
+	# Always build and install the kernel version found locally
 	info "Install kernel from sources"
 	pushd "${tmp_dir}" >> /dev/null
-	"${kernel_repo_dir}/kernel/build-kernel.sh" "setup"
-	"${kernel_repo_dir}/kernel/build-kernel.sh" "build"
-	sudo -E PATH="$PATH" "${kernel_repo_dir}/kernel/build-kernel.sh" "install"
+	"${kernel_repo_dir}/kernel/build-kernel.sh" -v "${kernel_version}" "setup"
```

Comment:
I know this is a cherry pick to a stable backport, but whilst reading it I noticed that this func is using the value of `kernel_version`, which is a var that seems to be declared and used in multiple functions in this file - I suspect either:
 - we should make that an explicit global var
 - or we should make the ones already present explicitly function local vars, and make them all call the version getting func.

Reply:
+1. Will do a follow-on cleanup PR around all this.